src/main/java/org/scoverage/plugin/SCoveragePreCompileMojo.java

```diff
@@ -212,6 +212,9 @@ public void execute() throws MojoExecutionException
 
             Properties projectProperties = project.getProperties();
 
+            // for  maven-compiler-plugin (compile), scala-maven-plugin (ompile)
```

Comment:
should be `compile` instead of `ompile`, right?

Reply:
Sure, I would correct it